integration-tests-ex/cases/cluster/Common/dependencies.yaml

```diff
@@ -86,7 +86,37 @@ services:
       MYSQL_USER: druid
       MYSQL_PASSWORD: diurd
 
-## TODO: Not yet retested
+  minio:
+    container_name: minio
+    command: server /data --console-address ":9001"
+    networks:
+      druid-it-net:
+        ipv4_address: 172.172.172.5
+    image: minio/minio:latest
+    ports:
+      - '9000:9000'
+      - '9001:9001'
+    volumes:
+      - ${SHARED_DIR}/minio:/data
+    environment:
+      - MINIO_ROOT_USER=${AWS_ACCESS_KEY_ID}
+      - MINIO_ROOT_PASSWORD=${AWS_SECRET_ACCESS_KEY}
+
+  create_minio_buckets:
+    image: minio/mc
+    networks:
+      druid-it-net:
+        ipv4_address: 172.172.172.6
+    depends_on:
+      - minio
+    entrypoint: >
+      /bin/sh -c "
+      /usr/bin/mc alias set s3 http://minio:9000 ${AWS_ACCESS_KEY_ID} ${AWS_SECRET_ACCESS_KEY};
+      /usr/bin/mc mb s3/${DRUID_CLOUD_BUCKET};
+      /usr/bin/mc anonymous set public s3/${DRUID_CLOUD_BUCKET};
+      "
```

Comment:
To prevent race conditions, one of the Druid containers has to depend on this one. That can be done by adding a dependency on, say, the overlord. The tests check that OL is ready, and OL won't be ready (won't start running) until the above is done.

I'm hoping that dependencies indicate only run order and not "OK can't start if create_minio_buckets" isn't running. Please double-check.

Reply:
Added dependency on create_minio_buckets for overlord.
Yeah, here dependencies indicate only run order.